docs/guides/q-ctrl-performance-management.ipynb

```diff
@@ -173,14 +173,14 @@
    "outputs": [],
    "source": [
     "import numpy as np\n",
-    "from qiskit.circuit.library import IQP\n",
+    "from qiskit.circuit.library import iqp\n",
     "from qiskit.quantum_info import random_hermitian, SparsePauliOp\n",
     "\n",
     "n_qubits = 50\n",
     "\n",
     "# Generate a random circuit\n",
     "mat = np.real(random_hermitian(n_qubits, seed=1234))\n",
-    "circuit = IQP(mat)\n",
+    "circuit = iqp(mat)\n",
```

Comment:
this guide is also maintained externally like the qunova one. While I don't think the change would break anything, I would apply the same suggestion that Julien left in the qunova guide just to be safe.

Reply:
done in [274bafa](https://github.com/Qiskit/documentation/pull/2852/commits/274bafa2839edcc320b0503b7b9425bba81dc008)